XYZDrivers/src/main/java/com/xyzdrivers/repositories/ClaimsRepo.java

```diff
@@ -53,8 +78,23 @@ public void update(Claim model) throws RepositoryException {
     }
 
     @Override
-    public void delete(Claim model) throws RepositoryException {
-        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+    public Claim update(Claim claim) {
+      
+        Object[] parameters = {
+            claim.getDate().toString(),
+            claim.getAmount(),
+            claim.getReason(),
+            claim.getStatus(),
+            claim.getId()
+        };
+        
+        try {
+            this.sqlService.executeUpdateStatement("UPDATE claims SET \"date\"=?, \"amount\"=?, \"rationale\"=?, \"status\"=? WHERE \"id\"=?", parameters);
+        } catch (SQLException ex) {
+            Logger.getLogger(ClaimsRepo.class.getName()).log(Level.SEVERE, null, ex);
+        }
+        
+        return claim;
```

Comment:
I think here you're trying to override the update method in Repo, but it's a void and so can't have a return. Same goes for the update in MembersRepo, and the unsupported update in PaymentRepo.

Also I think you left the old unsupported update method in.